src/20240602_AaveV3Ethereum_GhoStewardUpdate/AaveV3Ethereum_GhoStewardUpdate_20240602.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {IGhoStewardV2} from './interfaces/IGhoStewardV2.sol';
+import {IGhoToken} from './interfaces/IGho.sol';
+import {IGsm} from './interfaces/IGSM.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+
+/**
+ * @title Gho Steward Update
+ * @author ACI
+ * - Discussion: https://governance.aave.com/t/arfc-gho-stewards/16466/11
+ */
+contract AaveV3Ethereum_GhoStewardUpdate_20240602 is IProposalGenericExecutor {
+  address public constant GHO_STEWARD = 0x138d0f32C4D2eD3d4eEB37B1A059e56aFde87AB5;
+
+  function execute() external {
+    // Give Risk admin role to the steward
+    AaveV3Ethereum.ACL_MANAGER.addRiskAdmin(GHO_STEWARD);
```

Comment:
@marczeller lets remove all the admin permissions given to the old gho steward as well

Reply:
Done